src/app/shared/components/modal/modal.component.html

```diff
@@ -1,3 +1,6 @@
 <div class="modal">
   the very best modal
```

Comment:
But where is place to paste modal content?

Reply:
we do not specified how it should be, I add few tag's